src/ext/java/org/opentripplanner/ext/legacygraphqlapi/datafetchers/LegacyGraphQLPlaceImpl.java

```diff
@@ -74,16 +76,29 @@
     };
   }
 
-  // TODO
   @Override
-  public DataFetcher<BikePark> bikePark() {
-    return environment -> null;
+  public DataFetcher<Object> bikePark() {
+    return this::getVehicleParking;
   }
 
-  // TODO
   @Override
   public DataFetcher<Object> carPark() {
-    return environment -> null;
+    return this::getVehicleParking;
+  }
+
+  private VehicleParking getVehicleParking(DataFetchingEnvironment environment) {
+    FeedScopedId vehiclePlaceId = getSource(environment).place.stopId;
```

Comment:
I don't think `stopId` is the correct field to use here. It should be vehicleParkId but that doesn't seem to exist.

Reply:
Yes, this would be fixed in a subsequent PR of this branch:
https://github.com/realCity/OpenTripPlanner/commits/feature/vehicle-parking-api